go/start/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"breakfast/app"
+	"context"
+	"log"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+
+	// Create the client object just once per process
+	c, err := client.Dial(client.Options{})
+	if err != nil {
+		log.Fatalln("unable to create Temporal client", err)
+	}
+	defer c.Close()
+
+	options := client.StartWorkflowOptions{
+		ID:        "trip-planning-workflow",
```

Comment:
By specifying an ID like this it means two of these workflows cannot be running at the same time. While fine for a sample, in a real app you would never do this. I would maybe append a UUID to the workflow ID or not pass one.

Reply:
added UUID. thanks!